int32[] motor_id_arr
MotorDriverCmd[] motor_cmd
